build.gradle

```diff
@@ -19,6 +19,7 @@
 buildscript {
   ext.kotlin_version = '1.5.31'
   ext.compose_version = '1.0.5'
+  ext.dokka = "org.jetbrains.dokka:dokka-gradle-plugin:1.5.31"
```

Comment:
we need to apply the plugin in the dokka gradle file, so created this here, so we can reference it in both places